src/service/components/pulseaudio.js

```diff
@@ -110,7 +113,7 @@ class Stream {
  * The Mixer class uses GNOME Shell's Gvc library to control the system volume
  * and offers a few convenience functions.
  */
-const Mixer = GObject.registerClass({
+const Mixer = !Gvc ? null : GObject.registerClass({
     GTypeName: 'GSConnectAudioMixer',
 }, class Mixer extends Gvc.MixerControl {
     _init(params) {
```

Comment:
Because, previously, `Mixer` would always be defined.  But now, the code can cause it to initialize as `null` if `Gvc` hasn't resolved...